src/common/utils/Utils.ts

```diff
@@ -51,6 +51,17 @@ export function getArraySize(array: string[]): number {
   return 0;
 }
 
+export function traverseObjectRecursively(obj: any, callback: (visitedObject: any) => void): void {
```

Comment:
would a more meaningful name be something along the lines of "invokeCallbackRecursively"?

Also, we already have the `traverse` method in utils, would it be possible to utilise that?